### PR TITLE
fold cluster coordinates

### PR DESCRIPTION
**Pull request by rybkinjr**
Needed for interfacing to quantum chemistry software.

### CODE REVIEW

File: src/input_cp2k_dft.F [hunk 1]
```diff
@@ -6331,6 +6331,14 @@ SUBROUTINE create_print_simple_grid(section)
       CALL section_add_keyword(print_key, keyword)
       CALL keyword_release(keyword)
 
+      CALL keyword_create(keyword, name="FOLD_COORD", &
+                          description="Activates printing folded coordinates corresponding "// &
+                          "to the simple grid. Used as imput for external programs.", &
```
Comment (oschuett) [inline src/input_cp2k_dft.F]: Typo: imput -> input

Comment (rybkinjr) [reply src/input_cp2k_dft.F]: Corrected.

Comment (oschuett) [reply src/input_cp2k_dft.F]: Looks like you have not yet pushed the corrected commit. Try:
```
git add  src/input_cp2k_dft.F
git commit --amend 
git push --force origin embedding
```

Comment (rybkinjr) [reply src/input_cp2k_dft.F]: Thanks! I was not aware of the --amend option.

**Comment by oschuett** [comment]
Looks good. Feel free to hit **Rebase and merge**.

**Pull request merged**